library/core/src/slice/mod.rs

```diff
@@ -3983,6 +3983,66 @@ impl<T> [T] {
     }
 }
 
+#[cfg(not(bootstrap))]
+#[lang = "slice_f32"]
+impl [f32] {
+    /// Sorts the slice of floats.
+    ///
+    /// This sort is in-place (i.e. does not allocate), *O*(*n* \* log(*n*)) worst-case, and uses
+    /// the ordering defined by [`f32::total_cmp`].
+    ///
+    /// # Current implementation
+    ///
+    /// This uses the same sorting algorithm as [`sort_unstable_by`].
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// #![feature(sort_floats)]
+    /// let mut v = [2.6, -5e-8, f32::NAN, 8.29, f32::INFINITY, -1, 0, f32::NEG_INFINITY, -0];
```

Comment:
In Rust don't you need some dots there, like -1.0, 0.0, etc?

Reply:
You do need the dots: https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=bad2c59ff28b5bd7a5f1cdfa9c27227e

Though `-1.`, `0.`, etc can suffice.

Reply:
Fixed.